<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTC</class>
 <widget class="QMainWindow" name="CTC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CTC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>86</y>
      <width>981</width>
      <height>411</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="previousButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-- Previous</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>510</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Next --&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1012</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTrack">
    <property name="title">
     <string>Track Controls</string>
    </property>
    <addaction name="actionOpen_Close_Track"/>
   </widget>
   <widget class="QMenu" name="menuSet_Authority">
    <property name="title">
     <string>Set Authority</string>
    </property>
    <addaction name="actionDispatch_Train"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionView_Train_Statuses"/>
    <addaction name="actionView_Red_Line_Status"/>
    <addaction name="actionView_Green_Line_Status"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionAutomatic"/>
   </widget>
   <widget class="QMenu" name="menuSchedule">
    <property name="title">
     <string>Schedule</string>
    </property>
    <addaction name="actionView"/>
    <addaction name="actionUpdate"/>
   </widget>
   <addaction name="menuTrack"/>
   <addaction name="menuSet_Authority"/>
   <addaction name="menuMode"/>
   <addaction name="menuSchedule"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionView_Train_Statuses">
   <property name="text">
    <string>View Train Statuses</string>
   </property>
  </action>
  <action name="actionDispatch_Train">
   <property name="text">
    <string>Dispatch Train</string>
   </property>
  </action>
  <action name="actionOpen_Close_Track">
   <property name="text">
    <string>Open/ Close Track</string>
   </property>
  </action>
  <action name="actionView_Red_Line_Status">
   <property name="text">
    <string>View Red Line Status</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionView_Green_Line_Status">
   <property name="text">
    <string>View Green Line Status</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionAutomatic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatic</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>View</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
